build-pipeline/pipeline.json

```diff
@@ -16,19 +16,19 @@
         {
           "Name": "aspnet-docker-linux-images",
           "Parameters": {
-            "PB.image-builder.path": "1.0*"
```

Comment:
I'm not sure this is the right change.  This parameter is used in two ways.  One, it instructs the image-builder tool which Dockerfile paths in the manifest to build.  Two, it gets passed along as the `Folder` parameter to the test script.  The image-builder usage needs the wildcard.

Reply:
Well, something needs to change. Travis hasn't been actually been building or testing our images since this was changed. Is VSTS still working correctly for linux images?

Reply:
Of course something needs to change, I'm just pointing out this is going to break VSTS 😄 VSTS is building correctly but not running tests correctly on linux.

Reply:
Tested locally. It looks like adding double quotes around the env variable prevents bash from expanding the wildcard